app/bundles/UserBundle/DependencyInjection/MauticUserExtension.php

```diff
@@ -18,5 +19,10 @@ public function load(array $configs, ContainerBuilder $container): void
     {
         $loader = new PhpFileLoader($container, new FileLocator(__DIR__.'/../Config'));
         $loader->load('services.php');
+
+        $samlEnabled = $container->getParameter('mautic.saml_enabled');
+        if (true !== $samlEnabled) {
+            $container->removeDefinition(SAMLSubscriber::class);
+        }
```

Comment:
```suggestion
        if (!$samlEnabled) {
            $container->removeDefinition(SAMLSubscriber::class);
        }
```

Reply:
The variable must be of a boolean type. If it's not true, then the SAML is not enabled.